### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by AlexRRR**
Currently we are at 99 in performance and 93 accessibility.
PR raises this to 100 performance and 98 accessibility.

- CLS fix for sidebar when loading by setting min-width
- Change COVID19 accordion colours to achieve higher contrast 
- Fetch the language from metadata (excel) and set it in the `<html lang="es">` attribute
- Order heading levels in source document

 
Fix https://fieitas.atlassian.net/browse/FIEITAS-79

Test URLs:
- Before: https://main--website--fieitas.hlx.live/es/camping
- After: https://accessibility--website--fieitas.hlx.live/es/camping

### CODE REVIEW

**Comment by AlexRRR** [comment]
Screenshot 2023-06-01 at 10 27 10
Screenshot or it didn't happen

File: scripts/scripts.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {
   waitForLCP,
 } from './lib-franklin.js';
 
-const LCP_BLOCKS = []; // add your LCP blocks to the list
+const LCP_BLOCKS = ['gallery']; // add your LCP blocks to the list
```
Comment (andresbott) [inline scripts/scripts.js:19]: do we add the carousel as well?

File: scripts/scripts.js [hunk 2]
```diff
@@ -66,7 +66,8 @@ export function decorateMain(main) {
  * @param {Element} doc The container element
  */
 async function loadEager(doc) {
-  // document.documentElement.lang = 'en';
+  const lang = getMetadata('lang');
```
Comment (andresbott) [inline scripts/scripts.js:69]: :100:

**Review by andresbott — APPROVED** [review]

File: scripts/scripts.js (hunk 1 above)
Comment (AlexRRR) [reply scripts/scripts.js:19]: actually thanks for spotting that, it was an experiment which made no difference at all. I'll remove it

**Pull request merged**